integration-tests/__tests__/__snapshots__/coverage_report.test.js.snap

```diff
@@ -6,21 +6,20 @@ File           |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
 ---------------|----------|----------|----------|----------|----------------|
 All files      |      100 |      100 |      100 |      100 |                |
  a             |      100 |      100 |      100 |      100 |                |
-  identical.js |      100 |      100 |      100 |      100 |                |
+  Identical.js |      100 |      100 |      100 |      100 |                |
  b             |      100 |      100 |      100 |      100 |                |
-  identical.js |      100 |      100 |      100 |      100 |                |
+  Identical.js |      100 |      100 |      100 |      100 |                |
 ---------------|----------|----------|----------|----------|----------------|
 "
 `;
 
 exports[`collects coverage only from multiple specified files 1`] = `
-"---------------|----------|----------|----------|----------|----------------|
-File           |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------------|----------|----------|----------|----------|----------------|
-All files      |      100 |      100 |      100 |      100 |                |
- other-file.js |      100 |      100 |      100 |      100 |                |
- setup.js      |      100 |      100 |      100 |      100 |                |
----------------|----------|----------|----------|----------|----------------|
+"----------|----------|----------|----------|----------|----------------|
+File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
+----------|----------|----------|----------|----------|----------------|
+All files |        0 |      100 |      100 |        0 |                |
+ setup.js |        0 |      100 |      100 |        0 |              8 |
```

Comment:
actually, this snapshot is now missing `OtherFile` after renaming it from `other-file` to `OtherFile`